bin/node/runtime/src/lib.rs

```diff
@@ -992,9 +992,19 @@ parameter_types! {
 	pub const DesiredRunnersUp: u32 = 7;
 	pub const MaxVoters: u32 = 10 * 1000;
 	pub const MaxCandidates: u32 = 1000;
+	pub const MaxMembers: u32 = 20;
+	pub const MaxRunnersUp: u32 = 15;
 	pub const ElectionsPhragmenPalletId: LockIdentifier = *b"phrelect";
 }
 
+impl scale_info::TypeInfo for MaxCandidates {
```

Comment:
not sure why this is needed?

Reply:
Because `MaxCandidates` is used inside the `Voter` struct where it bounds the `votes` field,
```rust
#[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, TypeInfo, MaxEncodedLen)]
pub struct Voter<AccountId, Balance, MaxCandidates: Get<u32>> {
	/// The members being backed.
	pub votes: BoundedVec<AccountId, MaxCandidates>,
``` 
And since the `Voter` is a storage item, it implements `TypeInfo`, so `MaxCandidates` also needs to implement `TypeInfo`. Or is there a workaround so we don't have to implement `TypeInfo` for `MaxCandidates`?

Reply:
Actually, we don't need `TypeInfo`, we should probably use `MAXIMUM_VOTE` instead.